vta/python/vta/top/graphpack.py

```diff
@@ -55,6 +57,46 @@ def _unpack_batch_channel(data, old_shape):
     return data
 
 
+def _const_shape_match(data, dshape, cfactor_out):
```

Comment:
I suggest that we add assertions on shape length in case we have a non 6 dimensional tensor.

Reply:
fixed.